src/pages/ExerciseDetails/Head/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { ExerciseBase } from 'components/Exercises/models/exerciseBase';
+import { Language } from 'components/Exercises/models/language';
+import useComponentVisible from 'components/Header/useComponentVisible';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './head.module.css';
+
+export interface HeadProp {
+    exercise: ExerciseBase
+    languages: Language[]
+    changeLanguage: (lang: Language) => void,
+    language: Language | undefined // language displayed in the head since it's not found in the translations
+}
+
+export const Head = ({exercise, languages, changeLanguage, language}: HeadProp) => {
+    // const [displayLanguageState, setdisplayLanguageState] = useState(false);
+        
+    const category = exercise.category.name;
+    const equipment = exercise.equipment.length !== 0 ? exercise.equipment[0].name : "No equipment";
```

Comment:
An exercise can have multitple equipments, we should just show them all

Reply:
ok